src/main/java/com/sda/games/checkers/model/Main.java

```diff
@@ -4,21 +4,22 @@
 
 public class Main {
 
-    public static void main(String[] args) {
+    private static Scanner scanner = new Scanner(System.in);
 
-        Game.run();
-        Scanner scanner = new Scanner(System.in);
+    public static void main(String[] args) throws Exception {
 
-//        System.out.println("Enter White Player name:");
-//        String whitePlayerName = scanner.nextLine();
-//        System.out.println("Enter Black Player name:");
-//        String blackPlayerName = scanner.nextLine();
-//        Player whitePlayer = new Player(1, whitePlayerName, true);
-//        Player blackPlayer = new Player(2, blackPlayerName, false);
-
-        Board board = new Board();
-        board.resetBoard();
-        board.printBoard(board);
+        int menuOption = Menu.printMenu();
```

Comment:
chyba to zla nazwa dla funkcji skoro oprocz printowania zwraca to co odpowiedzial uzytkownik :D